webf/lib/src/html/input.dart

```diff
@@ -110,8 +110,8 @@ mixin BaseInputElement on WidgetElement {
 
   @override
   void initState() {
-    _focusNode = FocusNode();
-    _focusNode.addListener(() {
+    _focusNode ??= FocusNode();
```

Comment:
The following LateError was thrown building WebFContext:
    LateInitializationError: Field '_focusNode@1285021449' has not
    been initialized.
    
    The relevant error-causing widget was:
      WebFContext
      WebFContext:file:///Users/11095194/as_workspace/vmix-webf2/webf/webf/lib/src/widget/webf.dart:220:14
    
    When the exception was thrown, this was the stack:
    #0      BaseInputElement._focusNode (package:webf/src/html/input.dart)
    #1      BaseInputElement.initState (package:webf/src/html/input.dart:113:5)

Reply:
the late has been removed in this patch

Reply:
ok